Contributing-code-to-Oppia.md

```diff
@@ -8,16 +8,16 @@ If you run into any problems along the way, we're here to help! Check out our [[
 
 ## Table of Contents
 
-* [Setting things up](#setting-things-up)
-* [Developing your skills](#developing-your-skills)
-* [Finding something to do](#finding-something-to-do)
-  * [Good first issues for new contributors](#good-first-issues-for-new-contributors)
-    * [Backend](#backend)
-    * [Frontend](#frontend)
-  * [Contributor Roles](#contributor-roles)
-  * [Tasks for Existing Contributors](#tasks-for-existing-contributors)
-* [Tips for Success](#tips-for-success)
-* [Notes](#notes)
+- [Table of Contents](#table-of-contents)
+- [Setting things up](#setting-things-up)
+- [Developing your skills](#developing-your-skills)
+- [Finding something to do](#finding-something-to-do)
+  - [Good first issues for new contributors](#good-first-issues-for-new-contributors)
+  - [Choosing a good first issue](#choosing-a-good-first-issue)
+  - [Contributor Roles](#contributor-roles)
+  - [Tasks for Existing Contributors](#tasks-for-existing-contributors)
+- [Tips for Success](#tips-for-success)
+- [Notes](#notes)
 
```

Comment:
Hey @seanlip , I am not sure what these changes are about, I did not make any of these changes. Would you have any idea?

Reply:
I'm not sure either. Might be a bad merge; feel free to just undo them so that "Files Changed" looks correct.